packages/lambdas/src/document-uploader.ts

```diff
@@ -40,10 +42,15 @@ export const handler = async (event: S3Event) => {
         });
       }
     } catch (error) {
-      console.log("Error in documentUploaderHandler", error);
-      capture.error(error, {
-        extra: { context: `documentUploaderHandler`, sourceBucket, sourceKey, error },
+      const message = "Error in documentUploaderHandler";
+      console.log(message, error);
```

Comment:
Since we're here, let's use `errorToString`, please? We don't want to pass error as a second param to console.log because it results in multi-line logs that break on cloudwatch

Reply:
done